### DISCUSSION TITLE
How can change the color of the highlight indentChar related to the cursor?

### DISCUSSION THREAD

**Discussion by bassamsdata** [rose-pine/neovim#119] [Q&A] [answered]
First, thanks so much for this amazing theme.

I'm trying to change the `fg` color for the `indent-blankline` when I move the cursor. Now it change to slightly bright color because now it's really subtle and I don't see it to be honest.

Current situation:
![ezgif-2-98bf90a9f2](https://user-images.githubusercontent.com/105807570/213943291-60610004-40ce-4626-943a-97fe68082b7f.gif)

What I want:
![ezgif-2-fdda451278](https://user-images.githubusercontent.com/105807570/213943298-d523012b-fad7-4fdc-aea6-7d4964ebf9de.gif)

I tried these settings but no luck:

```lua
config = function()
      require("rose-pine").setup({
        disable_background = true,
        disable_float_background = false,
        highlight_groups = {
          IndentBlanklineContextChar = {
            fg = "#c5001c",
            nocombine = true,
          },
        },
      })
    end,
```

Thanks a lot

**Comment by mvllow** [comment, marked as answer]
I'm unable to reproduce this. I tried using main & canary branches with both a hex code for `fg` and a named Rosé Pine colour. Only difference I see between our configs is that I'm setting `vim.cmd.colorscheme` directly after running the setup.

```lua
return {
	{
		"lukas-reineke/indent-blankline.nvim",
		opts = {
			show_current_context = true,
			show_current_context_start = true,
		},
	},
	{
		"rose-pine/neovim",
		name = "rose-pine",
		-- I personally use canary but main branch also seemed fine
		branch = "canary",
		lazy = false,
		priority = 1000,
		config = function()
			require("rose-pine").setup({
				disable_italics = true,
				disable_float_background = true,
				highlight_groups = {
					IndentBlanklineContextChar = {
						fg = "iris",
						-- Tried with & without this, didn't seem to make a difference
						-- nocombine = true
					},
				},
			})
			vim.cmd.colorscheme("rose-pine")
		end,
	},
}
```

Maybe try setting colorscheme directly after and let me know if that works for you. Happy to help debug further if not – feel free to link to your config or create an issue with a minimal reproduction :)

**Reply by bassamsdata** [reply]
Thank you for your answer, really appreciated!

unfortunately, it's the same issue, whatever the color I change, nothing happens.

![ezgif-5-25bdb80cb7](https://user-images.githubusercontent.com/105807570/213953974-6c6fccca-3b2c-435d-97f8-c5479cdfdeee.gif)

My config is the [LazyVim starter](https://github.com/LazyVim/LazyVim), I haven't changed much in the settings.

thank you again.

edited: typo

**Reply by bassamsdata** [reply]
I even deleted the plugin and installed it from the canary branch and updated Lazy, quit nvim and opened many times, changed the color scheme and also changed to `vim.cmd.colorscheme("rose-pine")`. but it's the same output.

**Reply by mvllow** [reply]
I'll try to clone that config and see if I can reproduce but in the meantime maybe try updating Neovim to the latest nightly if not already.

We'll figure this out 😌

**Reply by mvllow** [reply]
Well this has been a doozy! So it looks like the highlight group you were using is only available if you enable the relevant option in indent-blankline (see below). After enabling that, however, the behaviour did not seem ideal. It actually looks like the group you want is `Delimiter`. I couldn't find this mentioned anywhere in the indent-blankline codebase so honestly there may be a better solution. I cloned the default LazyVim starter and created a new file at `lua/plugins/repro.lua` with the following contents:

```lua
return {
  {
    "lukas-reineke/indent-blankline.nvim",
    opts = {
      -- Uncomment this if you want to use the IndentBlanklineContextChar group below
      -- show_current_context = true,
    },
  },
  {
    "rose-pine/neovim",
    branch = "canary",
    lazy = true,
    name = "rose-pine",
    opts = {
      highlight_groups = {
        Delimiter = { fg = "iris" },
        -- IndentBlanklineContextChar = { fg = "iris" },
      },
    },
  },
  { "LazyVim/LazyVim", opts = { colorscheme = "rose-pine" } },
}
```

Hope this helps!

**Reply by bassamsdata** [reply]
Dude Big Big thanks to you. wow amazing 🔥.  the `Delimiter = { fg = "iris" }`, did the work. amazing.

I apologize for bothering you with this, but I really appreciate it. 🙏
